stable/rookout/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- MickaelAlliel
+- itielshwartz
+- LiranLast
+- Zohar2506
+reviewers:
+- MickaelAlliel
+- itielshwartz
+- LiranLast
+- Zohar2506
```

Comment:
Each of these people will need to follow the process at https://github.com/helm/charts#owning-and-maintaining-a-chart. Of the 4 people listed there are a total of 2 commits to charts.

Also, each person should be listed as a maintainer in the `Chart.yaml`.